openspec/changes/remaining-debt-clearance/tasks.md

```
@@ -35,26 +35,26 @@ that continue to work after the rename — the `*_test.ts` glob simply matches n
   git mv apps/api/src/modules/follow/index_test.ts apps/api/src/modules/follow/index.test.ts
   ```
 
-- [ ] 2.2 Search for any import references to the old filenames:
+- [x] 2.2 Search for any import references to the old filenames:
   `rg "cursor_test|service_test|model_test|index_test" apps packages -g '*.ts' -g '*.tsx'`
   Update any matches to the new names. (Test files are rarely imported, but verify.)
 
-- [ ] 2.3 In `AGENTS.md`, in the "Testing" section, add after the existing "Tests run with
+- [x] 2.3 In `AGENTS.md`, in the "Testing" section, add after the existing "Tests run with
   `--no-check`" line:
   ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)